### DISCUSSION TITLE
2FA Mobile

### DISCUSSION THREAD

**Discussion by xauuzd** [community/community#7011] [Mobile] [answered]
2FA Mobile does not work. It either does not work in the current country I'm in or takes extremely long/doesn't work. [I live in the UAE]

**Comment by brianlovin** [comment, marked as answer]
Hey @Pronner thanks for the heads up — what are you using as your 2FA method right now? It might be worth looking into change the 2FA method so that you don't need a network to deliver the code. Something like 1Password, or Google Authenticator can have the verification code instantly.

Let me know if you've tried that and things still aren't working.

**Reply by xauuzd** [reply]
It's alright now, I fixed it a while ago.

**Comment by candyho** [comment]
Marking this as answer since this is fixed.
